### DISCUSSION TITLE
 47c8ae6  Continuous build special Event Types 5

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1228] [General]
![4a779b886e381580e37c63deb09e101](https://github.com/artisan-roaster-scope/artisan/assets/116416332/38ea1d2a-84dc-4e96-86a7-07a43ecfb9b2)

**Comment by Terracotta-62** [comment]
Automatic marking TP offset

**Reply by MAKOMO** [reply]
Could you please share this .alog profile as well as the Artisan settings to reproduce this? Thanks a lot!

**Comment by Terracotta-62** [comment]
[Uploading 23-07-10_0921.zip…]()

**Reply by Terracotta-62** [reply]
The alog file already contains configuration information。

**Reply by MAKOMO** [reply]
Nothing attached. Did you drag-and-drop the zip file on a dialog window here on Github? The zip link just points back to the issue.

Artisan configuration contains your smoothing settings which might be relevant here. The .alog file does not contain those settings.

**Reply by MAKOMO** [reply]
If you feel unease to attach your settings here, then please send them to info@artisan.plus.

I did not understand the title of that issue either. What is wrong with Special Event Types 5 (untyped events I guess)? Looks all good to me.

**Reply by MAKOMO** [reply]
Got it. Thanks!

**Reply by Terracotta-62** [reply]
d58ce8f Continuous build  is perfect .  Thank you very much !

**Reply by roasterdave** [reply]
Thanks for reporting.  In a very rare circumstance it was possible for the TP of a background profile to be marked in the wrong location.  This is fixed in the continuous build, commit 5104c1d8 .

**Reply by Terracotta-62** [reply]
It has been verified to be true. Thanks!

**Comment by Terracotta-62** [comment]
I don't understand exactly how DUTY is configured. Do I need to refer to mathematical formulas or symbols?
Thanks！
![b5e9c9bd61d8c504509411792ff76d8](https://github.com/artisan-roaster-scope/artisan/assets/116416332/833eefe4-196c-410b-b312-bebcb92a17d2)

**Reply by MAKOMO** [reply]
The SV/Duty device type reports those values from the internal PID while it is active. In the future, please don't hijack discussion threads from another topic and rather open a new one with an adequate title to make it possible for others to find this information. Thanks!
